Java Script/index.js

```diff
@@ -0,0 +1,81 @@
+/*Loop for adding all the numbers bettwen 0 and 1000 with break > 400*/
+let num = 0;
+let sum = 0;
+for (num = 0; num < 1000; num++) {
+    sum += +num;
```

Comment:
Seems like there's no need for `+num` 🤔 
`sum += num`  is necessary, because it's doing `sum = sum + num`
but `+num` ideally converts num into "number" type...although you already defined num as "number".